contrib/mpi-proxy-split/record-replay.h

```diff
@@ -184,6 +186,16 @@ namespace dmtcp_mpi
     {
       return *(MPI_User_function**)_data;
     }
+
+    operator MPI_Comm_copy_attr_function*() const
+    {
+      return *(MPI_Comm_copy_attr_function**)_data;
+    }
+
+    operator MPI_Comm_delete_attr_function*() const
+    {
+      return *(MPI_Comm_delete_attr_function**)_data;
```

Comment:
These two function pointers are used in MPI_Comm_create_keyval. Could you add a test case for MPI_Comm_create_keyval to test them?